helm/charts/hpe-csi-driver/README.md

```diff
@@ -87,30 +70,21 @@ helm upgrade hpe-csi hpe/hpe-csi-driver --namespace kube-system --version=x.x.x.
 
 ### Uninstalling the Chart
 
-To uninstall (Helm 3) or delete (Helm 2) the `hpe-csi` chart:
+To uninstall the `hpe-csi` chart:
 
 ```
-# Helm 3
 helm uninstall hpe-csi --namespace kube-system
-
-# Helm 2
-helm delete hpe-csi --purge
 ```
 
-> **Note**: Due to a limitation in Helm, CRDs are not deleted as part of the chart uninstall.
+**Note**: Due to a limitation in Helm, CRDs are not deleted as part of the chart uninstall.
 
 ### Alternative install method
 
 In some cases it's more practical to provide the local configuration via the `helm` CLI directly. Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. These will take precedence over entries in [values.yaml](https://github.com/hpe-storage/co-deployments/blob/master/helm/values/csi-driver). For example:
 
 ```
-# Helm 3
-helm install hpe-csi hpe/hpe-csi-driver --namespace kube-system --set backendType=nimble \
---set backend=X.X.X.X --set username=admin --set password=xxxxxxxxx
-
-# Helm 2
-helm install --name hpe-csi hpe/hpe-csi-driver --namespace kube-system --set backendType=nimble \
---set backend=X.X.X.X --set username=admin --set password=xxxxxxxxx
+helm install hpe-csi hpe/hpe-csi-driver --namespace kube-system --set iscsi.chapUsername=admin \
+--set iscsi.chapPassword=xxxxxxxx
 ```
 
 ## Using persistent storage for Kubernetes
```

Comment:
Can we rewrite this with:

Enable dynamic provisioning of persistent storage by creating a `StorageClass` API object that references a `Secret` which maps to a supported HPE primary storage backend. Refer to the [HPE CSI Driver for Kubernetes](https://scod.hpedev.io/csi_driver/using.html) documentation on [HPE Storage Container Orchestration Documentation](https://scod.hpedev.io/). Also, it's helpful to be familiar with [persistent storage concepts](https://kubernetes.io/docs/concepts/storage/volumes/) in Kubernetes prior to deploying stateful workloads.